i18n/locales/en.json

```
@@ -102,6 +102,9 @@
         "approved": "Approved",
         "rejected": "Rejected"
     },
+    "modDashboardTopBar": {
+        "forReview": "For Review"
+    },
```

```suggestion
    "modDashboardTopBar": {
        "approved": "Approved"
        "forReview": "For Review",
        "rejected":  "Rejected"
    },
```

Just suggest adding these for now for when we later add the functionality for the label to change on the top

---

Changed applied. Thank you.